packages/stream/create-stream/create-stream.test.js

```diff
@@ -0,0 +1,238 @@
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream slug with not a string type', async function () {
+      const config = { name: 'shopping', slug: 2 }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to create a new stream because the slug of the stream is not in string format. A slug must be in string format'
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream description with not a string type', async function () {
+      const config = { name: 'shopping', slug: 'shopping', description: 2 }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to create a new stream because the description of the stream is not in string format. A description must be in string format'
+        )
+      }
+    })
+
+    it('should return error response if API Key is not valid', async function () {
+      nock(`${Internal.config.api.base_url}`)
+        .post(`/${Internal.config.api.version}/streams/create`, {
+          name: 'tes',
+          slug: 'tes',
+          description: 'tes',
+        })
+        .reply(403, { code: 403, message: 'API Key is not valid', data: '' })
+
+      const config = { name: 'shopping' }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to create a new stream because the API Key is not valid. Please provide a valid and active API Key.'
+        )
+      }
+    })
+
+    it('should return error response if unexpected error from server', async function () {
+      nock(`${Internal.config.api.base_url}`)
+        .post(`/${Internal.config.api.version}/streams/create`, {
+          name: 'tes',
+          slug: 'tes',
+          description: 'tes',
+        })
+        .reply(500, { code: 500, message: 'Server error', data: '' })
+
+      const config = { name: 'shopping' }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
```

Comment:
Same with the [comment on this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12/files#r973824303). We shouldn't use `try catch` for testing